docs/content/installation/from-source.zh-cn.md

```diff
@@ -52,7 +52,7 @@ git checkout v@version@
 
 - `go` @minGoVersion@ 或以上版本, 详见[这里](https://golang.google.cn/doc/install)
 - `node` @minNodeVersion@ 或以上版本，并且安装 `npm`, 详见[这里](https://nodejs.org/zh-cn/download/)
-- `make`, 详见[这里](/zh-cn/hacking-on-gitea/)
+- `make`, 详见[这里](/zh-cn/development/hacking-on-gitea/)
```

Comment:
```suggestion
- `make`, 详见[这里](development/hacking-on-gitea.md)
```

Reply:
https://docs.gitea.com/development/hacking-on-gitea.md is incorrect URL

Reply:
The link will be converted

Reply:
https://docs.gitea.com/development/hacking-on-gitea.md is incorrect URL
https://docs.gitea.com/development/hacking-on-gitea is correct URL
Right?